src/views/body.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @var string $id
+ * @var array $params
+ */
+
+?>
+
+<!-- Google Tag Manager (noscript) -->
+<noscript>
+    <iframe src="https://www.googletagmanager.com/ns.html?id=<?= $id ?>"
```

Comment:
я бы чисто на всякий случай сделал 
```suggestion
    <iframe src="https://www.googletagmanager.com/ns.html?id=<?= Html::encode($id) ?>"
```